lib/puppet/provider/package/pip.rb

```diff
@@ -31,7 +31,11 @@ def self.instances
     packages = []
     pip_cmd = self.pip_cmd
     return [] unless pip_cmd
-    execpipe "#{pip_cmd} freeze" do |process|
+    command = [pip_cmd, 'freeze']
+    if Puppet::Util::Package.versioncmp(self.pip_version, '8.1.0') >= 0 # a >= b
+        command << '--all'
```

Comment:
OK I'm lost. After I happily implemented the minor nitpick, the Appveyor says there is a conflict but does not specify against witch branch there is a conflict. This is pretty confusing.